pages/common/composer.md

```diff
@@ -0,0 +1,23 @@
+# composer
+
+> A package-based dependency manager for PHP projects.
+
+- Add a package as a dependency for this project, adding it to `composer.json`:
+
+`composer require {{user/package-name}}`
+
+- Install all dependencies in `composer.json` for this project:
```

Comment:
Just a nitpick, but I'd say "Install all dependencies in this project's `composer.json`" would be clearer (both here and in the second-to-last example in the page). What do you think?